src/core/EdFi.DataManagementService.Core/Handler/UpdateByIdHandler.cs

```diff
@@ -20,6 +23,11 @@ namespace EdFi.DataManagementService.Core.Handler;
 internal class UpdateByIdHandler(IDocumentStoreRepository _documentStoreRepository, ILogger _logger)
     : IPipelineStep
 {
+    private static string ToResourcePath(PathComponents p, DocumentUuid u)
```

Comment:
Maybe this belongs as a static method on PathComponents? Not an instance method, since documentUuid comes from a different place.

Reply:
moved.